UnityProject/Assets/Scripts/Hacking/HackingProcesses/HackingProcessDoorSimple.cs

```diff
@@ -0,0 +1,214 @@
+﻿using Mirror;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+/// <summary>
+/// This handles opening a network tab internally, and hence, doesn't require a HasNetworkTab component. Don't add one!
+/// </summary>
+public class HackingProcessDoorSimple : HackingProcessBase
+{
+	public NetTabType NetTabType = NetTabType.HackingPanel;
+
+	[SerializeField]
+	[Tooltip("The name that comes up when you interact with the object.")]
+	private string doorName = "airlock";
+
+	[SerializeField]
+	[Tooltip("SpriteRender which is toggled on when the door panel is exposed.")]
+	private SpriteRenderer hackPanelOverlay = null;
+
+	[SerializeField]
+	private Sprite hackPanelSprite = null;
+
+	private static int? seed = null;
+
+	private List<HackingNode> inputNodes = new List<HackingNode>();
+	private List<HackingNode> outputNodes = new List<HackingNode>();
+
+	private DoorController controller;
+	public DoorController Controller
+	{
+		get
+		{
+			if (!controller)
```

Comment:
```suggestion
			if (controller != null)
```
Makes intentions more clear.